charts/alfresco-repository/templates/deployment.yaml

```diff
@@ -68,6 +68,10 @@ spec:
             {{- $searchsecret = coalesce .search.existingSecret.name (include "alfresco-repository.fullname" $searchsecretCtx) }}
           {{- end }}
           env:
+            - name: ENC_METADATA_STOREPASS
+              value: mp6yc0UD9e
+            - name: ENC_METADATA_KEYPASS
+              value: oKIWzVdEdA
```

Comment:
these get overridden by the latter defined when using existingSecret right?

Reply:
indeed. One needs to pass the followig yaml to override these values:

```yaml
  configuration:
    repository:
      existingSecrets:
        - name: repository-keystores
          key: ENC_METADATA_STOREPASS
        - name: repository-keystores
          key: ENC_METADATA_KEYPASS
```